package.json

```diff
@@ -58,6 +58,8 @@
     "redux-action-buffer": "^1.1.0",
     "redux-logger": "^3.0.1",
     "redux-persist": "^4.8.2",
+    "redux-persist-transform-filter": "^0.0.10",
+    "redux-persist-transform-immutable": "^4.1.0",
```

Comment:
redux-persist-transform-immutable is redundant will remove it, don't remember how it got here!